src/routes/health_check.route.ts

```diff
@@ -1,8 +1,9 @@
 import { Router } from "express"
-import { healthCheckController } from "../controllers/health_check.controller";
+import { HealthCheckController } from "@/controllers";
 
 const router = Router();
+const healthCheckController = new HealthCheckController
 
-router.get("/health", healthCheckController);
+router.get("/health-check", healthCheckController.getStatus.bind(healthCheckController));
```

Comment:
why are you binding..is there an actual issue or you are just binding for the sake of binding??